html/clients/escootie.njk

```diff
@@ -6,6 +6,12 @@
 
 {% include 'components/header.njk' %}
 
+<script type="application/ld+json">
+    {
+        "image": "<%= pageUrl %>{{ pageImage }}?v<%= version %>",
```

Comment:
I am not sure if just picture will be enough for social media but other than that the versioned image URL is correct.

Reply:
I see, I'll add other attributes for Social medias as well

Reply:
However, I think I'll send another PR for this.